From 928d099e48a2c1496e1c2b049c385ebf455ca589 Mon Sep 17 00:00:00 2001
From: Jonathan DE CESCO <jonathan.de.cesco@intel.com>
Date: Wed, 24 Oct 2012 17:37:43 +0200
Subject: [PORT FROM MAIN] wifi: ease debugging by changing pr_debug with pr_info

BZ: 64298

With dynamic debug, it is not convenient to get the driver traces
displayed in the kmsg

Change-Id: Iddce5e540eaab71471da931fac82c43d1ccd54ce
Orig-Change-Id: Icfe68f74cb756652d07a715cecf573e742bc1217
Signed-off-by: Jonathan DE CESCO <jonathan.de.cesco@intel.com>
Reviewed-on: http://android.intel.com:8080/71522
Reviewed-by: buildbot <buildbot@intel.com>
Reviewed-by: Casasnovas, QuentinX <quentinx.casasnovas@intel.com>
Tested-by: Champciaux, Nicolas <nicolas.champciaux@intel.com>
Reviewed-by: lab_cactus <lab_cactus@intel.com>
Tested-by: lab_cactus <lab_cactus@intel.com>
Reviewed-on: http://android.intel.com:8080/78002
Reviewed-by: Li, XiaojingX <xiaojingx.li@intel.com>
Tested-by: Li, XiaojingX <xiaojingx.li@intel.com>
---
 wl12xx-compat/drivers/net/wireless/wl12xx/debug.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/wl12xx-compat/drivers/net/wireless/wl12xx/debug.h b/wl12xx-compat/drivers/net/wireless/wl12xx/debug.h
index e62fc3c..4c25db9 100644
--- a/wl12xx-compat/drivers/net/wireless/wl12xx/debug.h
+++ b/wl12xx-compat/drivers/net/wireless/wl12xx/debug.h
@@ -76,7 +76,7 @@ extern u32 wl12xx_debug_level;
 #define wl1271_debug(level, fmt, arg...) \
 	do { \
 		if (level & wl12xx_debug_level) \
-			pr_debug(DRIVER_PREFIX fmt "\n", ##arg); \
+			pr_info(DRIVER_PREFIX fmt "\n", ##arg); \
 	} while (0)
 
 /* TODO: use pr_debug_hex_dump when it becomes available */
-- 
1.7.4.1

